teemaderegister-be/src/controllers/auth.js

```diff
@@ -8,6 +8,16 @@ const { signToken, blacklistToken } = require('../utils/jwt')
 
 const { Error, InsertToken } = require('../utils/errors')
 
+module.exports.googleLogin = async (req, res) => {
```

Comment:
how is this endpoint authenticated? It should accept google auth secret to allow user actually get the token. Currenly anyone can get user token if they know email.